tutorials/NWB Tutorials/read_in_NWB.m

```diff
@@ -0,0 +1,49 @@
+%% *MatNWB Setup*
+% Start by setting up your MATLAB workspace. The code below adds the directory 
+% containing the MatNWB package to the MATLAB search path. MatNWB works by automatically 
+% creating API classes based on a defined schema. Running the generateCore() function
+% generates these classes for the lastest schema version.
+
+path_to_matnwb = '~/Repositories/matnwb';% change to your own path location
```

Comment:
we could include getting the MatNWB repo: 
```bash
% !git clone https://github.com/NeurodataWithoutBorders/matnwb.git
```

Reply:
Please let me know once you believe the tutorial is in its final form and ready for integration. I am very happy to merge it to the main repo. Thanks!